eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -452,8 +452,8 @@ rdptr ::
   ScriptPurpose era ->
   StrictMaybe RdmrPtr
 rdptr txBody = \case
-  Minting (PolicyID hash) ->
-    RdmrPtr Mint <$> indexOf hash (txBody ^. mintedTxBodyF :: Set (ScriptHash (EraCrypto era)))
+  Minting hash ->
+    RdmrPtr Mint <$> indexOf hash (txBody ^. mintedTxBodyF :: Set (PolicyID (EraCrypto era)))
```

Comment:
```suggestion
  Minting pId ->
    RdmrPtr Mint <$> indexOf pId (txBody ^. mintedTxBodyF :: Set (PolicyID (EraCrypto era)))
```

Reply:
:) I have a subsequent PR that does this rename

This implementation will be moved to a different module. So I'll keep this one as-is to avoid dealing with conflicts in another PR